Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -307,6 +307,7 @@ int ActsExamples::Sequencer::run() {
               StopWatch sw(localClocksAlgorithms[ialgo++]);
               ACTS_VERBOSE("Execute algorithm: " << alg->name());
               if (alg->execute(++context) != ProcessCode::SUCCESS) {
+                ACTS_ERROR("Failed to execute algorithm: " << alg->name());
```

Comment:
This could even be FATAL I guess.
```suggestion
                ACTS_FATAL("Failed to execute algorithm: " << alg->name());
```